docs/src/main/mdoc/dsl.md

```diff
@@ -419,6 +419,41 @@ val req = GET(uri"/weather/temperature/2016-11-05")
 dailyWeatherService.orNotFound(req).unsafeRunSync()
 ```
 
+### Handling matrix path parameters
+
+[Matrix path parameters](https://www.w3.org/DesignIssues/MatrixURIs.html) can be extracted using `MatrixVar`.
+
+```scala mdoc:silent
+import org.http4s.dsl.impl.MatrixVar
```

Comment:
Now that I see this, I wish we didn't have to import an `impl` package, but I think your doc is correct.